sale_order_type_automation/README.rst

```diff
@@ -0,0 +1,83 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Sale Order Type Automation
+==========================
+
+Add to kind of sales orders, options for when the sale is confirmed:
+    * The invoice is validated
+    * Payment of the invoice is validated and registrer
+    * The products are tranfer
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Just install
+
+
+Configuration
+=============
+
+To configure this module, you need to:
```

Comment:
se está repitiendo el título, dejar algo tipo:
To configure this module, you need to set validation options on sale order type:
(y luego las tres opciones que pusieron como items con #.)